core/scss/ng/card.scss

```diff
@@ -18,23 +18,23 @@
       }
     }
 
-    .md-card-layout {
-      &.md-card-block {
+    .md-ng1-card-layout {
+      &.md-ng1-card-block {
```

Comment:
I think we only need to add ng1 to the top levels. This will reduce adoption time for html users as well.

Reply:
Good idea

.md-ng1-card {
   &--block

Reply:
made the change on the top .. not touching the rest as I have no way to test this change